src/interface.js

```diff
@@ -0,0 +1,122 @@
+const guardianAPIKey = 'INSERT_YOUR_API_KEY_HERE';
+const guardianUrl = 'https://content.guardianapis.com/search?show-fields=all&api-key='+ guardianAPIKey;
+const aylienurl = 'http://news-summary-api.herokuapp.com/aylien?apiRequestUrl=https://api.aylien.com/api/v1/summarize?url=';
+const newsCollection = new NewsCollection();
+const date = new Date();
+const d = new Intl.DateTimeFormat('en-US').format(date);
+document.getElementById('date').innerHTML = d;
+
+
+window.onload = function() {
+  hideContent('full_news');
+  getNews();
+};
+
+
+window.onhashchange = function() {
+  const newsUrl = getUrl();
+  if (newsUrl == null || !newsUrl) {
+    getNews();
+  } else {
+    getSummary(newsUrl);
+  }
+};
+
+function getSummary(newsUrl) {
+  hideContent('news_collection');
```

Comment:
I really like the overall design - hiding the single article summary and then unhiding it (and hiding the multiple article summary) on a hashchange. All the refactoring you have done makes it very readable and so, even though I can't see it in action, I can see fairly quickly how it works. Well done!